macOS/LocalPackages/DataBrokerProtection/Sources/DataBrokerProtection/macOS/UserNotifications/DataBrokerProtectionUserNotificationService.swift

```diff
@@ -110,9 +110,9 @@ public class DefaultDataBrokerProtectionUserNotificationService: NSObject, DataB
         userNotificationCenter.add(request) { error in
             if error == nil {
                 if days != nil {
-                    Logger.dataBrokerProtection.log("Notification scheduled")
+                    Logger.dataBrokerProtection.log("PIR user notification scheduled")
```

Comment:
Just making some log statements a little clearer for my own benefit, since "notification" is a very generic term.